sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -13,7 +11,7 @@ if [[ ${PV} == 9999 ]]; then
 	# Use ~arch instead of empty keywords for compatibility with cros-workon
 	KEYWORDS="~amd64 ~arm64 ~arm ~x86"
 else
-	CROS_WORKON_COMMIT="0a7a9494e1427adfae12396716dea7903c875405"
+	CROS_WORKON_COMMIT="95adafc428b5b4be0ddd4d43a7b96658390388bc" # v229
```

Comment:
Could we cherry-pick this fix as well?
https://github.com/systemd/systemd/pull/2816

This is not critical for CoreOS but it would be nice to have: it would allow rkt's default stage1 to work on Fedora without having to disable SELinux.